sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -1,45 +1,466 @@
 # Azure Metrics Advisor client library for .NET
 
-Forthcoming
+Azure Cognitive Services Metrics Advisor is a cloud service that uses machine learning to monitor and detect anomalies in time series data. It includes the following capabilities:
 
-## Getting started
+- Analyze multi-dimensional data from multiple data sources.
+- Identify and correlate anomalies.
+- Configure and fine-tune the anomaly detection model used on your data.
+- Diagnose anomalies and help with root cause analysis.
+
+[Source code][metricsadv_client_src] | Package (NuGet) | [API reference documentation][metricsadv_refdocs] | Product documentation | [Samples][metricsadv_samples]
```

Comment:
Let's add the link to `metricsadv_nuget_package` here.

Reply:
Link product docs to here https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor/

Reply:
Just got a bit confused. `API reference documentation` was the one I should have removed.

Reply:
Problem was the `en-us` bit. Fixed.